### ISSUE TITLE
No version key

### ISSUE THREAD

**Issue by mbriney** [burnnat/ha-hdhomerun#11]
Getting the following warning that you need to add versioning to the integration in the future or HA will not load the files.

No 'version' key in the manifest file for custom integration 'hdhomerun'. As of Home Assistant 2021.6, this integration will no longer be loaded. Please report this to the maintainer of 'hdhomerun'

**Comment by curt7000** [comment]
yep, this integration breaks in 2021.6.  can you please update @burnnat ?

**Comment by jjanderson** [comment]
i recon this is dead.... :-(

**Comment by qwserty** [comment]
You can fix the problem by adding.

"version": "0.0.7",

to the manifest file in /config/custom_components/hdhomerun/manifest.json

**Comment by jjanderson** [comment]
I prefer not having unmaintained packages…

**Comment by Nils154** [comment]
This is fixed here: https://github.com/Nils154/ha-hdhomerun
